pkg/target/target.go

```diff
@@ -303,7 +307,14 @@ func addClusterLabels(opts *fleet.BundleDeploymentOptions, labels map[string]str
 		return err
 	}
 
-	opts.Helm.Values.Data = data.MergeMaps(opts.Helm.Values.Data, newValues)
+	if opts.Helm.DisablePreProcess == false {
+		opts.Helm.Values.Data, err = processTemplateValues(opts.Helm.Values.Data, values)
+		if err != nil {
+			return err
+		}
+		logrus.Debugf("preProcess completed for %v", opts.Helm.ReleaseName)
+	}
+
```

Comment:
When opts.Helm.DisablePreProcess = true, what will be set to opts.Helm.Values.Data? Seems like originally we were setting it using `opts.Helm.Values.Data = data.MergeMaps(opts.Helm.Values.Data, newValues)`

Reply:
opts.Helm.Values.Data is left alone in that case. It will either be a copy of the original from the deep copy or an empty map. https://github.com/rancher/fleet/blob/8af675f78e02838aed87fd2ddd16298a5e1960e8/pkg/target/target.go#L298-L304

But as I stare at it now that is dropping the global.fleet.clusterLabels from the helm values and should be added back.

Reply:
Seems like this is resolved?